README.md

```diff
@@ -302,6 +302,19 @@ yotiClient.getActivityDetails(token).then((activityDetails) => {
   * [X] Base64 Selfie Uri `getBase64SelfieUri()`
```

Comment:
Could you please also change the `const userProfile = activityDetails.getUserProfile(); // deprecated` from the start of the API coverage section above to use `getProfile()` instead of `getUserProfile()`?

Reply:
I've now updated this